src/main/java/io/github/aquerr/steamwebapiclient/request/PlayerRecentlyPlayedGamesRequest.java

```diff
@@ -0,0 +1,46 @@
+package io.github.aquerr.steamwebapiclient.request;
+
+import io.github.aquerr.steamwebapiclient.annotation.SteamRequestQueryParam;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * The request representing <a href="https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames">https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames</a>
+ */
+@Builder
+@Getter
+@Setter
+@ToString
+public class PlayerRecentlyPlayedGamesRequest implements SteamWebApiRestrictedRequest {
+
+        /**
+         * Steamworks Web API user authentication key.
+         *
+         * Added automatically by the client library if not added manually.
+         */
+        @SteamRequestQueryParam("key")
+        private String key = "";
+
+        /**
+         * Steam user id to be asked for data.
+         */
+        @SteamRequestQueryParam("steamid")
+        private long steamId;
+
+        /**
+         * Limit of results to be returned. Value above 5 always returns max 5 recent games.
```

Comment:
Let's get rid of that second meaning as I have just tested it myself and I got 16 games. Don't really know how this works but the first meaning alone "Limit of results to be returned." should be fine.